web/src/compose_validate.js

```diff
@@ -15,16 +15,33 @@ import {$t} from "./i18n";
 import {page_params} from "./page_params";
 import * as peer_data from "./peer_data";
 import * as people from "./people";
+import * as rows from "./rows";
 import * as settings_config from "./settings_config";
 import * as settings_data from "./settings_data";
 import * as stream_data from "./stream_data";
 import * as util from "./util";
 
 let user_acknowledged_wildcard = false;
+let current_editing_row;
 let wildcard_mention;
 
 export let wildcard_mention_large_stream_threshold = 15;
 
+export function set_current_editing_row(row) {
```

Comment:
While `upload.js` saves the row in the closure, I'm not certain if I can create a similar system within the given timeframe. Therefore, we need to find a solution to determine the appropriate location to append the banner. This seems to be the main problem that needs to be resolved.

Reply:
I really don't think we need this variable. 

Calls like `compose_validate.warn_if_mentioning_unsubscribed_user` can use `$textbox`, defined at the top of `content_typeahead_selected`, and I think those and the uploads are the main code paths outside the compose box that call into the `compose_validate` module at all.